### DISCUSSION TITLE
some features I would wish for

### DISCUSSION THREAD

**Discussion by CazadoraX** [stoatchat/discussions#415] [Feature Suggestions]
Here are some features I would love to see in Revolt:

> Gif Banner
> Calendar
> Forum Channel
> Self asignable roles
> roles shown in your servers Bio
> Crop Emojis, Icons, Pfp, Banner etc
> clock (so you can show the servers or users time)

**Comment by mdl2934** [comment]
to add to this

Channel categories like what guilded has:

- Calendar
- Scheduling
- Announcements
- Forum
- Stream
- Documents
- Media

[More here](https://support.guilded.gg/hc/en-us/sections/360012637333-Channels)

**Comment by ZHANGTIANYAO1** [comment]
+1
